components/Intro.tsx

```diff
@@ -1,30 +1,62 @@
 import config from "config";
+import { useDispatch } from "react-redux";
+import { setAdmin } from "redux/slices/admin";
+import { useAppSelector } from "redux/typesHooks";
 
-const Intro = () => (
-  <section className="intro-section">
-    <div className="container">
-      <div className="intro-content">
-        <div className="intro-avatar">
-          <div className="image">
-            <img
-              src="/images/author-avatar.jpg"
-              alt="author-avatar"
-              width={90}
-              height={90}
-            />
+const Intro = () => {
+  const dispatch = useDispatch();
+
+  const admin = useAppSelector((state) => state.admin);
+
+  const handleClick = () => {
+    if (admin === true) {
+      dispatch(setAdmin(false));
+    } else {
+      dispatch(setAdmin(true));
+    }
+  };
+
+  return (
+    <section className="intro-section">
+      <div className="container">
+        <div className="intro-content">
+          <div className="intro-avatar">
+            <div
+              className="image"
+              style={{ border: admin ? "3px solid #fe6c0a" : "none" }}
+            >
+              <div className="wrapper">
+                <img
+                  onDoubleClick={handleClick}
+                  src="/images/author-avatar.jpg"
+                  alt="author-avatar"
+                  width={90}
+                  height={90}
```

Comment:
```suggestion
                  width="90"
                  height="90"
```
Curly braces, are mostly used for variables placeholders, or dynamic calculations